ios/FluentUI.Demo/FluentUI.Demo/Demos/SideBarDemoController.swift

```diff
@@ -0,0 +1,122 @@
+
+	@objc private func presentSideBar() {
+		let controller = UIViewController(nibName: nil, bundle: nil)
+		contentViewController = controller
+		controller.modalPresentationStyle = .fullScreen
+		controller.view.backgroundColor = Colors.background1
+
+		sideBar = SideBar(frame: .zero)
+		sideBar!.insert(in: controller.view)
+		sideBar!.delegate = self
+
+		sideBar!.topItems = [
+			TabBarItem(title: "Home", image: UIImage(named: "Home_28")!, selectedImage: UIImage(named: "Home_Selected_28")!, landscapeImage: UIImage(named: "Home_24")!, landscapeSelectedImage: UIImage(named: "Home_Selected_24")!),
+			TabBarItem(title: "New", image: UIImage(named: "New_28")!, selectedImage: UIImage(named: "New_Selected_28")!, landscapeImage: UIImage(named: "New_24")!, landscapeSelectedImage: UIImage(named: "New_Selected_24")!),
+			TabBarItem(title: "Open", image: UIImage(named: "Open_28")!, selectedImage: UIImage(named: "Open_Selected_28")!, landscapeImage: UIImage(named: "Open_24")!, landscapeSelectedImage: UIImage(named: "Open_Selected_24")!)
+		]
+
+		sideBar!.bottomItems = [
+			TabBarItem(title: "Help", image: UIImage(named: "Help_28")!, selectedImage: UIImage(named: "Help_Selected_28")!, landscapeImage: UIImage(named: "Help_24")!, landscapeSelectedImage: UIImage(named: "Help_Selected_24")!),
+			TabBarItem(title: "Settings", image: UIImage(named: "Settings_28")!, selectedImage: UIImage(named: "Settings_Selected_28")!, landscapeImage: UIImage(named: "Settings_24")!, landscapeSelectedImage: UIImage(named: "Settings_Selected_24")!)
+		]
+
+		let optionsStackView = UIStackView(frame: .zero)
+		optionsStackView.axis = .vertical
+		optionsStackView.alignment = .center
+		optionsStackView.spacing = 5.0
+		optionsStackView.translatesAutoresizingMaskIntoConstraints = false
+		controller.view.addSubview(optionsStackView)
+
+		let showAvatarViewRow = createLabelAndSwitchRow(labelText: "Show Avatar View", switchAction: #selector(toggleAvatarView(switchView:)), isOn: true)
+		showAvatarViewRow.translatesAutoresizingMaskIntoConstraints = false
+		optionsStackView.addArrangedSubview(showAvatarViewRow)
+		showAvatarView(true)
+
+		let button = Button()
+		button.translatesAutoresizingMaskIntoConstraints = false
+		button.titleLabel?.textAlignment = .center
+		button.titleLabel?.numberOfLines = 0
+		button.setTitle("Dismiss", for: .normal)
+		button.addTarget(self, action: #selector(dismissSideBar), for: .touchUpInside)
+		optionsStackView.addArrangedSubview(button)
+
+		NSLayoutConstraint.activate([
+			optionsStackView.centerXAnchor.constraint(equalTo: controller.view.centerXAnchor),
+			optionsStackView.centerYAnchor.constraint(equalTo: controller.view.centerYAnchor),
+		])
+
+		present(controller, animated: false)
+	}
+
+	private func createLabelAndSwitchRow(labelText: String, switchAction: Selector, isOn: Bool = false) -> UIView {
+		let stackView = UIStackView(frame: .zero)
+		stackView.axis = .horizontal
+		stackView.alignment = .center
+		stackView.spacing = 10.0
```

Comment:
Even though this is test code, should still split constants out to the bottom to set a good example for consumers who are using this as a template